catkit/testbed/experiment.py

```diff
@@ -64,7 +104,9 @@ def __init__(self, safety_tests, safety_check_interval=60, output_path=None, suf
         self.post_experiment_return = None
 
         self.safety_check_interval = safety_check_interval
-        self.safety_tests = safety_tests.copy()
+        self.safety_tests = []
+        for test in safety_tests:
+            self.safety_tests.append(test())
```

Comment:
I reverted this back to how it was in hicat when I remembered that it was like this such that only this instance can own and have access to the safety tests.